packages/frontend/src/hooks/chat/useDraft.ts

```diff
@@ -492,6 +503,105 @@ export function useDraft(
     debouncedSaveAndRefetchDraft,
   ])
 
+  const handleSetDraftRequest = useCallback(() => {
```

Comment:
```suggestion

  /**
   * Handle `window.__setDraftRequest` which might have been set
   * before the chat was loaded to set a draft with specified text and/or file.
   */
  const handleSetDraftRequest = useCallback(() => {
```
A comment here would help to understand what this function does without reading all code